salesagility/SuiteCRM
# Fixed #217 AOD Ignores unified_search_modules_display

Add conditional to include Unified_search_module in AOD_Index.php
## Description

Create variable where can retrieve data from UnifiedSearchAdvanced.php
and add condition into isModuleSearchable.


Go to admin/global search
In list global search take note of enabled modules for searching
Go back to Home screen
Type searched string into global search box and click loop icon
If the searched data come from enabled modules it should display result
Referenced bug #217


## Motivation and Context


## How To Test This

Go to admin/global search
In list global search move some modules from enabled to disabled modules for searching
Go back to Home screen
Type searched string into global search box and click loop icon
If the searched data come from enabled modules it should display result
## Types of changes


### Final checklist
